Podfile

```diff
@@ -1,7 +1,11 @@
 platform :ios, '8.0'
-
 source 'https://github.com/CocoaPods/Specs.git'
+use_frameworks!
+
+target 'CTAssetsPickerDemo' do
+  pod 'CTAssetsPickerController', :path => '.'
```

Comment:
I suspect this was necessary during testing right? If so please revert this as I don't want to break existing CocoaPods support.

Reply:
This is necessary for all pods since several years. It's just a new way of writing pod file. If you run pod init on new project, you will see target statement in new podfile. It does not break existing cocoapod.

<img width="944" alt="Screenshot 2020-04-22 at 08 39 15" src="https://user-images.githubusercontent.com/3957568/79945042-ad320d00-8475-11ea-8362-d905a8ed4792.png">